docs/admin/troubleshooting.md

```diff
@@ -919,6 +919,45 @@ pool header correct
 0
 ```
 
+## Syslog
+
+RAS events are printed to the Syslog by 'daos_server' processes via the golang standard library API.
+If no Syslog daemon is configured on the host, errors will be printed to the 'daos_server' log file:
+
+```
+ERROR: failed to create syslogger with priority %!s(syslog.Priority=24): Unix syslog delivery error
+```
+
+These errors will not prevent DAOS from running as expected but indicate that no Syslog messages
+will be delivered.
```

Comment:
I forget. Does this also mean that the RAS events will effectively be blackholed? i.e. Do we always print them to the control log and also syslog? I have a recollection that we decided not to do that because it seemed too spammy. If that's the case, it might be worth pointing out in this documentation that RAS events may be lost if syslog isn't available.

We may also want to include a link to the section on RAS events so that the reader can understand what they are and why they're important.

Reply:
```
131 >       out := evt.PrintRAS()
132 >       if sl := el.sysloggers[evt.Severity]; sl != nil {
133 >       >       sl.Print(out)
134 >       >       return
135 >       }
136 >       // no syslogger, write to default logger instead
137 >       el.log.Info("&&& RAS " + out)
138 }
```
Will print to control log if syslog is unavailable so the entries should not be lost.

Added link to RAS events section.